src/selic-entries.ts

```diff
@@ -0,0 +1,11 @@
+export async function selicEntries(initialDate: string, finalDate: string) {
+    const url = new URL('https://api.bcb.gov.br/dados/serie/bcdata.sgs.11/dados?formato=json');
+    const params = new URLSearchParams;
+    params.append('dataInicial', initialDate);
+    params.append('dataFinal', finalDate);
```

Comment:
You can also assign an object to be taken as the parameters.

```suggestion
    const params = new URLSearchParams(
        dataInicial: initialDate,
        dataFinal: finalDate,
    );
```